maraboupy/MarabouNetworkNNet.py

```diff
@@ -163,66 +377,305 @@ def variableRanges(self):
 
         for layer, hidden_layer_length in enumerate(hidden_layers):
             for i in range(hidden_layer_length):
-                offset = sum([x*2 for x in hidden_layers[:layer]])
+                offset = sum([x * 2 for x in hidden_layers[:layer]])
 
                 b_variables.append(self.layerSizes[0] + offset + i)
-                f_variables.append(self.layerSizes[0] + offset + i+hidden_layer_length)
+                f_variables.append(self.layerSizes[0] + offset + i + hidden_layer_length)
 
-        #final layer
+        # final layer
         for i in range(self.layerSizes[-1]):
-            offset = sum([x*2 for x in hidden_layers[:len(hidden_layers) - 1]])
-            output_variables.append(self.layerSizes[0] + offset + i + 2*hidden_layers[-1])
+            offset = sum([x * 2 for x in hidden_layers[:len(hidden_layers) - 1]])
+            output_variables.append(self.layerSizes[0] + offset + i + 2 * hidden_layers[-1])
 
         self.inputVars = np.array([input_variables])
         self.b_variables = b_variables
         self.f_variables = f_variables
         self.outputVars = np.array([output_variables])
 
     def nodeTo_b(self, layer, node):
-        """Compute variable number for the backward variable of a node in a layer
-
+        """Get variable number for the backward variable corresponding to layer, node
         Args:
             layer (int): Layer number
-            node (int): Node number
+            node (int): Node index within layer
 
         Returns:
             (int)
-        
-        :meta private:
+
+        :meta-private:
```

Comment:
When using the meta private tag, there shouldn't be a hyphen between meta and private. It should be just ":meta private:", otherwise the auto-documentation will look messed up.

I think this is the last comment I have. So before we merge we'll need to update this PR with changes made to the Marabou master branch to make sure travis builds correctly and the documentation looks correct. @guykatzz did you have any additional comments?

Reply:
The meta-private needs to be changed in nodeTo_f too.

Reply:
Done. Thanks!

@guykatzz, any more comments? Can we merge this?